openml-h2o/src/test/java/com/feedzai/openml/h2o/H2OModelProviderTrainTest.java

```diff
@@ -91,15 +91,16 @@ private Dataset createDataset(final DatasetSchema schema) {
      *
      * @param schema The schema that the dataset will comply to.
      * @param size   The number of instances of the dataset.
+     *
      * @return A new dataset.
      */
     private Dataset createDataset(final DatasetSchema schema, final int size) {
         final Random random = new Random(234);
 
         logger.info("Using dataset size of {}", size);
         final List<Instance> instances = IntStream.range(0, size)
-                .mapToObj(index -> new MockInstance(schema, random))
-                .collect(Collectors.toList());
+                                                  .mapToObj(index -> new MockInstance(schema, random))
```

Comment:
not sure if you added this space manually or not. But we have a IDEA settings, and we use IDEA auto format. This avoids polluting the PR with "useless" and subjective changes

Reply:
We is an over statement :D